hack/dockerfiles/license.Dockerfile

```diff
@@ -0,0 +1,47 @@
+# syntax=docker/dockerfile:1
+
+# Copyright 2022 BuildKit Syft scanner authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG LICENSE_HOLDER="BuildKit Syft scanner authors"
```

Comment:
I wonder if this should be `buildkit-syft-scanner`? Just to keep the name the same as the repo for clarity.